cmd/earthly/satellite_cmds.go

```diff
@@ -220,16 +229,86 @@ func (app *earthlyApp) useSatellite(cliCtx *cli.Context, satelliteName, orgName
 	return nil
 }
 
-func (app *earthlyApp) printSatellitesTable(satellites []cloud.SatelliteInstance, orgID string) {
+type satelliteWithPipelineInfo struct {
+	satellite cloud.SatelliteInstance
+	pipeline  *cloud.Pipeline
+}
+
+func (app *earthlyApp) toSatellitePipelineInfo(satellites []cloud.SatelliteInstance, pipelines []cloud.Pipeline) []satelliteWithPipelineInfo {
+	res := make([]satelliteWithPipelineInfo, 0)
+	for _, s := range satellites {
+		swp := satelliteWithPipelineInfo{
+			satellite: s,
+		}
+		for _, p := range pipelines {
+			if s.Name == p.SatelliteName {
+				swp.pipeline = &p
+				break
+			}
+		}
+		res = append(res, swp)
+	}
+	return res
+}
+
+func printRow(t *tabwriter.Writer, c []color.Attribute, items []string) {
+	sprint := color.New(c...).SprintFunc()
```

Comment:
this doesn't adhere to the `NO_COLOR` and `FORCE_COLOR` env options.

Reply:
Turns out it does, [its baked in to the color library](https://github.com/fatih/color#disableenable-color), and appeared to work? At least, for disabling it. Setting `color.NoColor = true` apparently is a global thing in this package. [We do that here](https://github.com/earthly/earthly/blob/9dad001d998e0a6a262ffbc7c2c7f5521cac311d/cmd/earthly/main.go#L286).

Reply:
> I used the toggle [baked in to the color library](https://github.com/fatih/color#disableenable-color), and appeared to work? At least, for disabling it.

cool!

maybe it's fine as-is, and then we can fix it later if a user reports it as an issue.

my worry is someone will want to pipe the output into something else, and then will have to deal with color chars (where as must programs will detect if it's an interactive tty and only use color in that case)